src/renderers/noop/ReactNoop.js

```diff
@@ -160,6 +172,14 @@ var ReactNoop = {
     }
   },
 
+  hasScheduledDeferredCallback() {
+    return Boolean(scheduledDeferredCallback);
```

Comment:
I'm open to suggestions! It doesn't seem as bad to me because we're testing API for renderers by writing a custom renderer. That's different from messing with internal modules from an existing renderer.